bedrock/settings/base.py

```diff
@@ -717,6 +717,13 @@ def get_app_name(hostname):
 POCKET_CONSUMER_KEY = config("POCKET_CONSUMER_KEY", default="")
 POCKET_ACCESS_TOKEN = config("POCKET_ACCESS_TOKEN", default="")
 
+BRAZE_API_URL_BASE = config("BRAZE_API_URL_BASE", default="https://rest.iad-05.braze.com")
+BRAZE_API_KEY = config("BRAZE_API_KEY", default="")
+BRAZE_API_NEWSLETTERS = {
+    "news": config("BRAZE_API_NEWSLETTER_ID_NEWS", default=""),
+    "hits": config("BRAZE_API_NEWSLETTER_ID_HITS", default=""),
```

Comment:
Is the idea for the new page to also replace https://getpocket.com/en/explore/pocket-hits-signup ?

Reply:
no. but the idea is for us to move that page here as well, so I'm just getting out ahead of that.